config/babel.config.js

```diff
@@ -33,9 +33,8 @@ module.exports = {
     i18n: {
       plugins: [
         [
-          'react-intl',
+          'formatjs',
           {
-            messagesDir: './temp/babel-plugin-react-intl',
             moduleSourceName: '@edx/frontend-platform/i18n',
```

Comment:
Looks like `moduleSourceName` was removed [in v9.0.0](https://github.com/formatjs/formatjs/blob/main/packages/babel-plugin-formatjs/CHANGELOG.md#900-2021-01-04).  Can you confirm we really don't need to do anything else in addition to just removing the line?

Reply:
yes this one seems extra, i think we can safely remove it